lib/futurism/helpers.rb

```diff
@@ -68,10 +68,18 @@ def initialize(extends:, placeholder:, options:)
         @eager = options.delete(:eager)
         @broadcast_each = options.delete(:broadcast_each)
         @controller = options.delete(:controller)
+        @updates_for_object = options.delete(:updates_for)
         @html_options = options.delete(:html_options) || {}
         @data_attributes = html_options.fetch(:data, {}).except(:sgid, :signed_params)
         @model = options.delete(:model)
+        @wrapped_for_updates_for = options.delete(:wrapped_for_updates_for)
+        if @wrapped_for_updates_for
+          @html_options[:keep] = 'keep'
+          @data_attributes['updates-for'] = true
```

Comment:
Instructing the javascript to listen for the CableReady `cable-ready:after-update` event. As we only want to refresh the partial if it has been rendered before.